### PR TITLE
style changes

### PR DESCRIPTION
**Pull request by robhyrk**

### CODE REVIEW

**Comment by maltekliemann** [comment]
Closed #39.

File: src/components/HomepageFeatures/index.tsx [hunk 1]
```diff
@@ -63,5 +70,21 @@ export default function HomepageFeatures(): JSX.Element {
         </div>
       </div>
     </section>
+    <section className={clsx('row', styles.homeCTA)}>
+      <div className={clsx('container', styles.homeCTAinner)}>
+        <div className={clsx('col col--6')}>
+            <h2>Help improve this wiki</h2>
+            <p>This wiki was started by and is maintained by Web3 Foundation. It is an open-source-ish project and aims to be the most extensive resource of knowledge on Polkadot and the Polkadot ecosystem.</p>
```
Comment (maltekliemann) [inline src/components/HomepageFeatures/index.tsx:77]:
```suggestion
            <h2>Help improve the Zeitgeist documentation</h2>
            <p>The Zeitgeist documentation is an open-source project maintainted by Zeitgeist PM and continuously improved by the Zeitgeist community.</p>
```

**Review by maltekliemann — CHANGES_REQUESTED** [review]
Hey @robhyrk! Again, many thanks for prettifying our site. Looks gorgeous! 💯 

I'm sure that you'll receive the new assets very soon and can start making the required changed. Other than that, I've noticed a couple of small things.

- The sidebar on the left get's cut off when scrolling down. This happens on the Polkadot wiki, as well, so I wouldn't be surprised if there's no immediate fix for this.
  Screenshot 2022-06-17 at 14 49 01

- Outside of the launch page, the Zeitgeist logo doesn't render correctly:
  Screenshot 2022-06-17 at 14 24 22

- The social media icons don't show anywhere. Seemed to be working yesterday. :shrug:

**Comment by robhyrk** [comment]
@maltekliemann thanks for the feedback. 

I couldn't replicate the scrolling issue so hopefully that got resolved when I updated the docs with the new assets; otherwise it might be an issue with Docusaraus as you mentioned the Polkdadot site also has it.

Logo in the footer should be fixed now too...was just missing a '/' when pointing to the asset.

Largest update was incorporating all the new design changes so it should look nice and sleek now.

**Review by maltekliemann — APPROVED** [review]
Looks good to me!

**Pull request merged**